packages/codemods/package.json

```
@@ -0,0 +1,51 @@
+{
+  "name": "@suspensive/codemods",
+  "version": "0.1.0",
+  "description": "Codemods for @suspensive.",
+  "keywords": [
+    "suspensive",
+    "codemods"
+  ],
+  "homepage": "https://suspensive.org",
+  "bugs": "https://github.com/toss/suspensive/issues",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/toss/suspensive.git",
+    "directory": "packages/codemods"
+  },
+  "license": "MIT",
+  "author": "Gwansik Kim <gwansik.kim@gwansik.dev> & Jonghyeon Ko <jonghyeon@toss.im>",
```

```suggestion
  "author": "Gwansik Kim <gwansik.kim@gwansik.dev>",
```
I will leave the authorship to you and rely on you to take full responsibility for all issues and responsibilities. I will only review and approve code to prevent dangerous things. This is not a choice to neglect you. I hope you will be clear about what I can and cannot help with and accept it as a way to speed up this project! I will support this project in a way that respects freedom as much as possible.

---

Sure! I’ll take on the responsibility and do my best! 🫡